packages/bpk-component-select/src/bpk-select.scss

```diff
@@ -40,4 +40,24 @@
       @include bpk-select--docked-last-child;
     }
   }
+
+  &--with-image {
+    display: flex;
+    padding-right: 0;
+    flex-direction: row;
+    align-items: center;
+    background: none;
+
+    .bpk-select {
+      height: auto;
+      padding-top: 0;
+      padding-bottom: 0;
+      border: transparent;
+    }
```

Comment:
This is clever but I think it's kind of contravening BEM (slightly). How come we changed away from doing it as a mixin?

Also - this is probably not a modifier (my bad), it's probably better as an element like `bpk-select-image-wrapper`.